From 452bfd970a9ed8c90aaabc84de000a44c6406177 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Tue, 4 Nov 2014 07:15:01 +0100
Subject: [PATCH] Update Linux-aarch64 test outputs

Related to #42
---
 test/outputs/Linux-aarch64/run_test.log.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/test/outputs/Linux-aarch64/run_test.log.in b/test/outputs/Linux-aarch64/run_test.log.in
index 3d83a66..4ec6544 100644
--- a/test/outputs/Linux-aarch64/run_test.log.in
+++ b/test/outputs/Linux-aarch64/run_test.log.in
@@ -1,4 +1,4 @@
-Caught exception java.io.FileNotFoundException in method java.io.FileInputStream.<init>()
+Caught exception java.io.FileNotFoundException in method java.io.FileInputStream.open()
 Exception in thread "main" java.io.FileNotFoundException: _wrong_file_ (No such file or directory)
 	at java.io.FileInputStream.open(Native Method) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileInputStream.class]
 	at java.io.FileInputStream.<init>(FileInputStream.java:LINENO) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileInputStream.class]
@@ -6,7 +6,7 @@ Exception in thread "main" java.io.FileNotFoundException: _wrong_file_ (No such
 	at Test.fileRelatedIssues(Test.java:461) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.main(Test.java:513) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
-Caught exception java.io.FileNotFoundException in method java.io.FileInputStream.<init>()
+Caught exception java.io.FileNotFoundException in method java.io.FileInputStream.open()
 Exception in thread "main" java.io.FileNotFoundException: /root/.bashrc (Permission denied)
 	at java.io.FileInputStream.open(Native Method) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileInputStream.class]
 	at java.io.FileInputStream.<init>(FileInputStream.java:LINENO) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileInputStream.class]
@@ -14,7 +14,7 @@ Exception in thread "main" java.io.FileNotFoundException: /root/.bashrc (Permiss
 	at Test.fileRelatedIssues(Test.java:462) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.main(Test.java:513) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
-Caught exception java.io.FileNotFoundException in method java.io.FileOutputStream.<init>()
+Caught exception java.io.FileNotFoundException in method java.io.FileOutputStream.open()
 Exception in thread "main" java.io.FileNotFoundException: /root/.bashrc (Permission denied)
 	at java.io.FileOutputStream.open(Native Method) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileOutputStream.class]
 	at java.io.FileOutputStream.<init>(FileOutputStream.java:LINENO) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/io/FileOutputStream.class]
@@ -23,7 +23,7 @@ Exception in thread "main" java.io.FileNotFoundException: /root/.bashrc (Permiss
 	at Test.fileRelatedIssues(Test.java:463) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.main(Test.java:513) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
-Caught exception java.net.UnknownHostException in method java.net.InetAddress$2.lookupAllHostAddr()
+Caught exception java.net.UnknownHostException in method java.net.Inet6AddressImpl.lookupAllHostAddr()
 Exception in thread "main" java.net.UnknownHostException: xyzzy: unknown error
 	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/net/Inet6AddressImpl.class]
 	at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:LINENO) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/net/InetAddress$2.class]
@@ -50,7 +50,7 @@ Exception in thread "main" java.net.UnknownHostException: xyzzy
 	at Test.networkRelatedIssues(Test.java:473) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 	at Test.main(Test.java:514) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
-Caught exception java.net.ConnectException in method java.net.AbstractPlainSocketImpl.doConnect()
+Caught exception java.net.ConnectException in method java.net.PlainSocketImpl.socketConnect()
 Exception in thread "main" java.net.ConnectException: Connection refused
 	at java.net.PlainSocketImpl.socketConnect(Native Method) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/net/PlainSocketImpl.class]
 	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:LINENO) [jar:file:JAVA_AND_SYSTEM_SPECIFIC_PATH/rt.jar!/java/net/AbstractPlainSocketImpl.class]
-- 
2.1.0

